Page/LoginPage.robot

```diff
@@ -3,12 +3,16 @@ Library           Selenium2Library
 Resource          ../globalConfig/testEnv.robot
 
 *** Variables ***
-${USER_NAME_INPUT_TEXT_LOCATOR}    j_username
-${PASSWORD_INPUT_TEXT_LOCATOR}    j_password
+${USER_INPUT_LOCATOR}    //input[@id='j_username']
+${PASSWORD_INPUT_LOCATOR}    //input[@name='j_password']
+${LOGIN_BUTTON_LOCATOR}    //button[contains(text(),'Login')]
 
 *** Keywords ***
 Login
-    Open Browser    ${CMP_ADRESS}
-    Input Text    ${USER_NAME_INPUT_TEXT_LOCATOR}    ${ADMIN_LOGIN}
-    Input Text    ${PASSWORD_INPUT_TEXT_LOCATOR}    ${ADMIN_PASSWORD}
+    [Arguments]    ${login}    ${password}
+    Input Text    ${USER_INPUT_LOCATOR}    ${login}
+    Input Text    ${PASSWORD_INPUT_LOCATOR}    ${password}
     Submit Form
+
+Wait For Login Page
+    Wait Until Element Is Enabled    ${USER_INPUT_LOCATOR}
```

Comment:
Timeout is missed. Do not hardcode the value of timeout. Create a constant in page.robot SELENIUM_DEFAULT_TIMEOUT and use it over the framework